src/encoded/commands/import_data.py

```diff
@@ -22,7 +22,7 @@
 """
 from webtest import TestApp
 from urllib.parse import urlparse
-from .. import loadxl
+from encoded import loadxl
```

Comment:
seems like an odd time clean this up; is this a python6+ issue?

Reply:
It got rid of some of the pytest warning.